### PR TITLE
Stringtie update

### PR DESCRIPTION
**Pull request by bgruening**

### CODE REVIEW

File: tools/stringtie/stringtie.xml [hunk 1]
```diff
@@ -122,7 +122,7 @@ stringtie "$input_bam"
             <output file="ballgown/t_data.ctab" ftype="tabular" name="transcript_expression" />
             <output file="ballgown/e2t.ctab" ftype="tabular" name="exon_transcript_mapping" />
             <output file="ballgown/i2t.ctab" ftype="tabular" name="intron_transcript_mapping" />
-            <output file="stringtie_out5.gtf" ftype="gtf" name="output_gtf" />
+            <output file="stringtie_out5.gtf" ftype="gtf" name="output_gtf" lines_diff="2" />
```
Comment (erasche) [inline tools/stringtie/stringtie.xml]: Any reason for all of these lines_diff?

Comment (bgruening) [inline tools/stringtie/stringtie.xml]: I saw a version number in the outputs and was not willing to update all the test data.

**Comment by davebx** [comment]
Enthusiastic :+1: from me.

File: tools/stringtie/stringtie.xml (hunk 1 above)
Comment (erasche) [inline tools/stringtie/stringtie.xml]: Works for me! :)

**Pull request merged**